src/thridParty/Tess4J/build.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
rename the folder from `thridParty` to `thirdParty`
Also move this folder at the same level of `src` folder

Reply:
done